### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by hydraa000** [HolographicWings/TOTK-Mods-collection#153] [Q&A]
will dynamic fps update come?

**Comment by hoverbike1** [comment]
Everybody is anxiously waiting for Chucks response.

**Comment by Blazovsky69** [comment]
When can we expect a general update for all the community mods for 1.2.1 ?

**Reply by hoverbike1** [reply]
NXCheats has updated the cheats, so they're covered.
Stevens has provided me with updates to most graphical mods.

However, without a version of DFPS that supports 1.2.1, these patches are not very useful.
So, we have to patiently wait for Chuck to finish updating DFPS.
Can always check here to see if he has been active recently:
https://www.reddit.com/user/ChucksFeedAndSeed/comments/

**Reply by Blazovsky69** [reply]
Thanks for the awnser !

Just tried the updated version of 60fps static with DFPS v1.5.3 on 1.2.1, and (I don't know if 60fps static is the best FPS mod among the other ones) it seems to be working really fine for the moment.

**Reply by hoverbike1** [reply]
With the release of 4.20, 1.2.1 is fully supported now.
